gateway-service/src/main/resources/application.properties

```diff
@@ -1,3 +1,3 @@
 server.port=8081
```

Comment:
Idea:

Let's leave the gateway at 8080 since that's the default port

And have 
user-service: 8081
budget: 8082
notification: 8083
email: 8084
calculation: 8085